app/controllers/application_controller.rb

```diff
@@ -1,6 +1,9 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+
+  skip_before_action :verify_authenticity_token
```

Comment:
This is due to JS-related errors, right?
I believe there is a possible fix along these lines (adding a header with a specific value): https://github.com/shakacode/react_on_rails/blob/c8bbb087c4760fdec86c0ccdd8f67045f93f0ce0/node_package/src/Authenticity.js